README.md

```diff
@@ -38,6 +45,14 @@ Brion provides the following major features:
   [Lunchbox](http://eyescale.github.io/Lunchbox-1.8/index.html),
   [Boost](http://www.boost.org/doc/libs).
 
+## Algorithmic library
```

Comment:
Not algorithmic yet. But it also depends on the low-level defintion of an algorithm ;)